src/rgw/rgw_common.h

```diff
@@ -1527,6 +1527,17 @@ struct rgw_obj_key {
     snprintf(buf, sizeof(buf), "%s[%s]", name.c_str(), instance.c_str());
     return buf;
   }
+
+  // Helper function to find the key of an object.
+  // in case of unversioned bucket object key is returned as is.
+  // in case of versioned bucket object key is key[version_id].
```

Comment:
Mention in the comment that this additional function is for handling the null version if specified in the request.

Reply:
Done